pmd-kotlin/src/main/java/net/sourceforge/pmd/lang/kotlin/KotlinLanguageModule.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.kotlin;
+
+import net.sourceforge.pmd.lang.BaseLanguageModule;
+
+/**
+ * Language Module for Kotlin
+ */
+public class KotlinLanguageModule extends BaseLanguageModule {
+
+    /** The name. */
+    public static final String NAME = "Kotlin";
+    /** The terse name. */
+    public static final String TERSE_NAME = "kotlin";
+
+    /**
+     * Create a new instance of Kotlin Language Module.
+     */
+    public KotlinLanguageModule() {
+        super(NAME, null, TERSE_NAME, "kt", "ktm");
+        addVersion("1.3", new KotlinHandler("1.3"));
+        addVersion("1.4", new KotlinHandler("1.4"));
+        addDefaultVersion("1.5", new KotlinHandler("1.5"));
```

Comment:
How to we handle updating the parser?
We probably can get rid of the version number, as we anyway don't check it like in java. For other languages, like Apex, we have only one version here and we update the version number to the latest supported language version. Which means, whenever we update the grammar, we also update the version here.

Reply:
For now, I've updated the grammar to the latest version and kept only one version in this list (1.6).